src/components/HomeSectionTestimonials/index.tsx

```diff
@@ -26,6 +30,10 @@ const HomeSectionTestimonials = () => (
       </BlockquoteFooter>
     </Blockquote>
     <Logos>
+      <Logo src={appsmithLogo} width="40" height="62" />
+      <Logo src={castorLogo} width="127" height="62" />
+      <Logo src={hugoLogo} width="40" height="62" />
+      <Logo src={sitepointLogo} width="127" height="62" />
```

Comment:
This should be equal to the original sizes in the SVGs, e.g.:

```
<svg width="273" height="62" viewBox="0 0 273 62" fill="none" xmlns="http://www.w3.org/2000/svg">
```

```suggestion
      <Logo src={appsmithLogo} width="60" height="62" />
      <Logo src={castorLogo} width="188" height="62" />
      <Logo src={hugoLogo} width="61" height="62" />
      <Logo src={sitepointLogo} width="273" height="62" />
```

Reply:
Would you also mind putting these logos below the old ones, please?

Reply:
Done :)